ccswbs/doorknob
# First Test of TailwindCSS

A first test of using TailwindCSS on the statistic component.

Note I have only updated the Gradient part of the statistic component as thats the only one used on doorknob

Feel free to take as much time reviewing this, this is more an exploration of a potential development tool and as such isn't a major priority.

# Summary of changes

## Frontend
- Added and updated various files to install TailwindCSS:
- Updated `statistic.js` component to use Tailwind rather than Styled Components

### Incidental changes
- N/A

## Backend
- N/A

## Test plan
- PR build: https://doorknobmain-tailwind.gatsbyjs.io/

1. View the statistic component on https://doorknobmain-tailwind.gatsbyjs.io/
1. Check it on various breakpoints, ensure it works as expected.
1. Try adding or removing statistics in a local environment.
1. View the code in `statistic.js` & `tailwind.config.js` and get an idea of how TailwindCSS works.
1. Read through the TailwindCSS documentation https://tailwindcss.com/docs/utility-first
1. Give feedback/provide thoughts on how you feel about TailwindCSS
